src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Colors getRandomColor() {
+        int colorIndex = new Random().nextInt(Colors.values().length);
```

Comment:
```suggestion
        int colorIndex = random.nextInt(Colors.values().length);
```
Create field inside ColorSupplier class for Random object;